Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/User/UserGroup/ByUserGroup.tsx

```diff
@@ -56,6 +56,8 @@ const ByUser: Application.Types.iByComponent = (props) => {
     const sortField = useSelector(SecurityGroupSlice.SortField);
     const ascending = useSelector(SecurityGroupSlice.Ascending);
 
+    const [filterableList, setFilterableList] = React.useState<Search.IField<ISecurityGroup>[]>(defaultSearchcols);
```

Comment:
Does this really need to be a state? If we never set it to anything but `defaultSearchcols` we can just use that

Reply:
That's true. I was referencing the way the Users page operates and was taking bits and pieces from that. I can change it to just use `defaultSearchcols`.